src/components/partials/IdentitySearch/IdentitySearch.vue

```diff
@@ -96,13 +97,21 @@ export default class IdentitySearch extends Vue {
     editCustomNodeModalOpen = false;
     customNode?: ChainInfo;
     chainOptions: UISelectOption[] = [];
+    keyPressed = false;
 
     created() {
         const searchParams = new URLSearchParams(window.location.search);
         this.searchTerm = searchParams.get("query") ?? "";
         this.selectedChainKey = searchParams.get("chain") ?? "";
         this.loadCustomNodeFromStorage();
         this.setChainOptions();
+        if (this.searchTerm) {
+            this.keyPressed = true;
+        }
```

Comment:
@sosan-neikbeen I didn't get that. If the searchTerm is set from the URL query, why are you setting ```keyPressed=true```?

I would try to fix the issue like so:
```html
<!-- The input w/o v-model -->
<input :value="searchTerm" @keyup="onInputKeyUp" ...>
```

```typescript
// on key up set the search term,
// that should enable the send button immediately
onInputKeyUp(event: KeyboardEvent) {
    this.searchTerm = event.target.value;
}
```

But that is just an idea, not sure if that works.

Reply:
This is the best way to do. Hopefully it would work. I cannot test this feature on my end. Because everything seems to work fine in desktop.